pkg/controller/component/component_controller.go

```diff
@@ -56,12 +56,15 @@ func add(mgr manager.Manager, r reconcile.Reconciler) error {
 	if err != nil {
 		return err
 	}
-
 	return nil
 }
 
-var _ reconcile.Reconciler = &ReconcileComponent{}
-var buildTypeImages = map[string]string{"nodejs":"nodeshift/centos7-s2i-nodejs:10.x"}
+
+var (
+	_                  reconcile.Reconciler = &ReconcileComponent{}
+	buildTypeImages                        = map[string]string{"nodejs": "nodeshift/centos7-s2i-nodejs:10.x"}
+	openshiftNamespace                      = "openshift"
```

Comment:
Let's move this to a different file. I'm expecting this list to grow to at 6-8 images.